docs/changelog-fragments/477.bugfix.rst

```diff
@@ -0,0 +1,2 @@
+Extended CFLAGS to include header locations typical to FreeBSD
+(:file:/usr/local/include) -- by :user:`donnerhacke`
```

Comment:
```suggestion
(:file:`/usr/local/include`) -- by :user:`donnerhacke`
```

Reply:
(missing such bits of syntax is exactly why it's best to accept the suggestions from GH UI)